source/common/res/features/budget-category-info/main.js

```diff
@@ -160,7 +160,8 @@
           // when this is moved to the webpack codebase, hopefully we can just listen to onRouteChanged
           // but until then we'll just have to keep adding stuff to this check that should trigger
           // budget-category related features to update
-          if (changedNodes.has('navlink-budget active') ||
```

Comment:
It looks like this may have worked in the past, but since then nYNAB has changed the programming such that these nodes are destroyed and then created, instead of simply changing the class of the existing nav item. Because of this, 'navlink-budget active' is not in the list of changed nodes.